docs/en/understand_mmcv/data_transform.md

```diff
@@ -0,0 +1,337 @@
+           # more introduction will be referred in the following chapters (random variable sharing)
+           share_random_params=True,
+           transforms=[
+               # only need to manipulate the "img" field in the `RandomFlip` class
+               dict(type='RandomFlip'),
+           ])
+   ]
+   ```
+
+   In the `mapping` setting of the multi-target extension, we can also use `...` to ignore the specified original field. As shown in the following example, the wrapped `RandomCrop` will crop the image in the field `"img"` and update the size of the cropped image if the field `"img_shape"` exists. If we want to do the same random cropping for both image fields `"lq"` and `"gt"` at the same time but update the `"img_shape"` field only once, we can do it as in the example:
+
+   ```python
+   pipeline = [
+       dict(type='TransformBroadcaster',
+           mapping={
+               'img': ['lq', 'gt'],
+               'img_shape': ['img_shape', ...],
+            },
+           # remap the "img" and "img_shape" fields back to their original fields after the transformation
+           auto_remap=True,
+           # whether to share random variables in the transformation of each target
+           # more introduction will be referred in the following chapters (random variable sharing)
+           share_random_params=True,
+           transforms=[
+               # "img" and "img_shape" fields are manipulated in the `RandomCrop` class
+               # if "img_shape" is missing, only operate on "img"
+               dict(type='RandomCrop'),
+           ])
+   ]
+   ```
+
+2. A set of targets applied to a field
+
+   Suppose we need to apply a data transformation to the `"images"` field, which is a list of images.
+
+   ```python
+   pipeline = [
+       dict(type='TransformBroadcaster',
+           # map each image under the "images" field to the "img" field
+           mapping={'img': 'images'},
+           # remap the images under the "img" field back to the list in the "images" field after the transformation
+           auto_remap=True,
+           # whether to share random variables in the transformation of each target
+           share_random_params=True,
+           transforms=[
+               # in the `RandomFlip` transformation class, we only need to manipulate the "img" field
+               dict(type='RandomFlip'),
+           ])
+   ]
+   ```
+
+#### Decorator `cache_randomness`
+
+In `TransformBroadcaster`, we provide the `share_random_params` option to support sharing random states across multiple data transformations. For example, in a super-resolution task, we want to apply random transformations **synchronously** to the low-resolution image and the original image. If we use this function in a custom data transformation class, we need to mark which random variables support sharing in the class. This can be achieved with the decorator `cache_randomness`.
```

Comment:
```suggestion
In `TransformBroadcaster`, we provide the `share_random_params` option to support sharing random states across multiple data transformations. For example, in a super-resolution task, we want to apply **the same** random transformations **simultaneously** to the low-resolution image and the original image. If we use this function in a custom data transformation class, we need to mark which random variables support sharing in the class. This can be achieved with the decorator `cache_randomness`.
```